deckhouse-controller/pkg/controller/module-controllers/utils/utils.go

```diff
@@ -15,10 +15,15 @@
 package utils
 
 import (
+	"crypto/md5"
+	"fmt"
+
 	"github.com/deckhouse/deckhouse/deckhouse-controller/pkg/apis/deckhouse.io/v1alpha1"
 	"github.com/deckhouse/deckhouse/go_lib/dependency/cr"
 )
 
+var ModuleRestartRequiredAnnotation = "modules.deckhouse.io/restart-required"
```

Comment:
What is module restart, and how can it be required? What will happen if a user sets this annotation on a module?

Reply:
it was decided not to have a go at the module restart functionality this time, so this code was removed